File: Source/Rio.Models/DataTransferObjects/User/UserDto.cs
using Rio.Models.DataTransferObjects.Role;
using System;

namespace Rio.Models.DataTransferObjects.User
{
    public class UserDto : UserSimpleDto
    {
        public Guid? UserGuid { get; set; }
        public string Phone { get; set; }
        public string LoginName { get; set; }
        public RoleDto Role { get; set; }
    }
}

File: Source/Rio.Web/src/app/shared/models/user/user-dto.ts
import { RoleDto } from '../role/role-dto';

export class UserDto {
    UserID: number;
    UserGuid: string;

    FirstName: string;
    LastName: string;
    FullName: string;

    Email: string;
    Phone: string;
    LoginName: string;

    Role: RoleDto;

    DisclaimerAcknowledgedDate: Date;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

